testassets/RepoThatShouldFailToBuild/src/BadConsole/BadConsole.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Internal.AspNetCore.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Will this change need to be made in all our repos?

Reply:
Yes, if there are any. I don't see anything using this pattern in the master branch anymore.